addons/website_crm/models/website_visitor.py

```diff
@@ -20,7 +20,7 @@ def _compute_email_phone(self):
         super(WebsiteVisitor, self)._compute_email_phone()
         self.flush()
 
-        left_visitors = self.filtered(lambda visitor: not visitor.email or not visitor.mobile)
+        left_visitors = self.sudo().filtered(lambda visitor: not visitor.email or not visitor.mobile)
```

Comment:
Instead of adding the `sudo` on this line, add it on the next line: only when you need it to read the leads. By default, you do not want every read on `left_visitors` afterwards to be done in `sudo` mode (I would have thought you could have wanted that but: https://discord.com/channels/678381219515465750/687339689522364423/1154041170452807791)